engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetBlending.java

```diff
@@ -24,13 +24,13 @@
 /**
  * TODO: Add javadocs
  */
-public final class SetBlend extends SetCapability {
+public class SetBlending extends SetCapability {
```

Comment:
Still recommending to rename SetCapability to SetStateParameter or perhaps an even more specific SetBooleanStateParameter.